tests/server/testServerWithStorage.js

```diff
@@ -186,10 +218,12 @@ sjrk.storyTelling.server.testServerWithStorageDefs = [{
         "onBlankStoryWithEmptyMediaBlocksSaveSuccessful": null
     },
     testUploadOptions: {
-        testFile: "./tests/testData/logo_small_fluid_vertical.png",
+        testPNGFile: "./tests/testData/logo_small_fluid_vertical.png",
+        testImageWithCorrectOrientation: "./tests/testData/obliterationroom.jpg",
```

Comment:
It might be helpful to rename images to "correctOrientation.jpg" and "incorrectOrientation.jpg".

Reply:
Good point! The `obliterationroom.jpg` image was already present, so rather than include a separate case to cover a correctly-oriented image, I decided to re-use that. Since that file is used in other tests, I think I'll find another image that has correct orientation and then call that `correctOrientation.jpg`